examples/python/example_vamana_dynamic.py

```diff
@@ -22,11 +22,12 @@
 # [imports]
 
 DEBUG_MODE = False
-def assert_equal(lhs, rhs, message: str = ""):
+def assert_equal(lhs, rhs, message: str = "", epsilon = 0.05):
     if DEBUG_MODE:
         print(f"{message}: {lhs} == {rhs}")
     else:
-        assert lhs == rhs, message
+        assert lhs < rhs + epsilon, f"{message}"
+        assert lhs > rhs - epsilon, f"{message}"
```

Comment:
Do we need the f-string?
```suggestion
        assert lhs < rhs + epsilon, message
        assert lhs > rhs - epsilon, message
```

Reply:
Good catch, fixed!